noxfile.py

```diff
@@ -8,7 +8,7 @@
 package = "modelldcatnotordf"
 locations = "src", "tests", "noxfile.py", "docs/conf.py"
 nox.options.stop_on_first_error = True
-nox.options.sessions = "lint", "mypy", "pytype", "tests"
+nox.options.sessions = "black", "lint", "mypy", "pytype", "tests"
```

Comment:
ikke ta med black i default nox

black endrer koden, det er ikke noe vi skal gjøre i løpet av en deploy

Reply:
Ok @NilsOveTen , da skal jeg fikse dette. :+1:

Reply:
Det var eg som la inn black. Eg har sett at ein god del kode kjem inn i master utan å vere igjennom black. Det bygger enorm "teknisk gjeld" og blir uhåndterbart over tid. (rdflib-folka slit med dette for eksempel.) Men eg er forsåvidt komfortabel med at dette heller blir ein del av review-prosessen.
Bottom line: kode som er merga skal ikkje generere black-diffs.